src/docs/tutorial-transaction-process/create-transaction-process/index.md

```diff
@@ -0,0 +1,411 @@
+
+Here's a screenshot of the transaction card in the Flex Console. It
+shows a transaction in the preauthorized state - the decline link for
+the marketplace operator is on the right-side column.
+
+![Operator's decline transition in Console.](./decline-preauthorized-by-operator.png)
+
+## Update client app
+
+After we have changed the transaction process, we also need to take the
+new process into use in our client app.
+
+In this tutorial, we assume that we don't need to care about ongoing
+transactions. It is important to consider this before taking a new
+process version into use. When a transaction is created, it is tied to
+the version of the process that was in use at that time. Therefore, you
+might need to update your client app, so that it supports several
+different process versions.
+
+### Update config.js
+
+```shell
+└── src
+    └── config.js
+```
+
+Since we have created a new process, we need to update
+**bookingProcessAlias** variable in config.js. This variable defines
+what transaction process is used when a new transaction is created from
+the web app.
+
+```js
+const bookingProcessAlias = 'cottagedays-daily-booking/release-1';
+```
+
+In the same file, we need to also update variable: **bookingUnitType**.
+It defines some UI changes (mostly about translations) - i.e. is this
+booking about _nights_, _days_, or some other _units_.
+
+```js
+const bookingUnitType = 'line-item/day';
+```
+
+### Update transaction.js
+
+The web app needs to understand how the connected transaction process
+works and, therefore, the process graph is actually duplicated in the
+current versions of FTW templates.
+
+```shell
+└── src
+    └── util
+        └── transaction.js
+```
+
+So, we need to make the transaction process change also there.
+
+**Step 1**: Create a new transition variable:
+`TRANSITION_DECLINE_BY_OPERATOR`.
+
+```js
+// When the provider accepts or declines a transaction from the
+// SalePage, it is transitioned with the accept or decline transition.
+export const TRANSITION_ACCEPT = 'transition/accept';
+export const TRANSITION_DECLINE = 'transition/decline';
+
+export const TRANSITION_DECLINE_BY_OPERATOR =
+  'transition/decline-preauthorized-by-operator';
+```
+
+**Step 2**: Create a new state variable: `STATE_DECLINED_BY_OPERATOR`.
+
+```js
+const STATE_DECLINED = 'declined';
+const STATE_DECLINED_BY_OPERATOR = 'declined-by-operator';
+```
+
+**Step 3**: Update the process graph.<br /> Variable _stateDescription_
+contains the same process graph in _Xstate_ format and we need to add
+the new state and transition there.
+
+```js
+    [STATE_PREAUTHORIZED]: {
+      on: {
+        [TRANSITION_DECLINE]: STATE_DECLINED,
+        [TRANSITION_DECLINE_BY_OPERATOR]: STATE_DECLINED_BY_OPERATOR,
+        [TRANSITION_EXPIRE]: STATE_DECLINED,
+        [TRANSITION_ACCEPT]: STATE_ACCEPTED,
+      },
+    },
+
+    [STATE_DECLINED]: {},
+    [STATE_DECLINED_BY_OPERATOR]: {},
+```
+
+**Note**: You could also update the id of the graph.
```

Comment:
I think this could be a separate step before the current step 3 or then inside step 3 but mentioned before updating the actual graph (because the id comes in the code first).

Reply:
It's not strictly speaking necessary to update it, but I'll mention this at the beginning of step 3.